kernel.go

```diff
@@ -244,8 +263,18 @@ func handleExecuteRequest(ir *classic.Interp, receipt msgReceipt) error {
 		ExecCounter++
 	}
 
+	// Prepare the map that will hold the reply content
```

Comment:
Period.  I will stop commenting on these, but you get the idea.  I would just scrub the comments.